random.c

```diff
@@ -497,7 +516,8 @@ fill_random_bytes_syscall(void *buf, size_t size, int unused)
 {
 #if (defined(__OpenBSD__) && OpenBSD >= 201411) || \
     (defined(__NetBSD__)  && __NetBSD_Version__ >= 700000000) || \
-    (defined(__FreeBSD__) && __FreeBSD_version >= 1200079)
+    (defined(__FreeBSD__) && __FreeBSD_version >= 1200079) || \
+    defined(__APPLE__)
```

Comment:
In what cases is this necessary?

Reply:
Maybe not ... too much randomness at a time in this case indeed :)

Reply:
If this condition is enabled,  this `fill_random_bytes_syscall()` never fails and `fill_random_bytes_urandom()` using `getentropy()` is never used.

Reply:
true I removed it in the last commit anyway.